Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx

```diff
@@ -375,7 +377,7 @@ void TrackInterpolation::extrapolateTrack(int iSeed)
     }
     TPCClusterResiduals res;
     res.setDY(y - trkWork.getY());
-    res.setDY(z - trkWork.getZ());
+    res.setDZ(z - trkWork.getZ());
```

Comment:
Had a stupid typo here for ITS-TPC tracks. This did not affect any global processing, as we did not use ITS-TPC only tracks for the residuals yet, but took me a bit to figure out now